src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,50 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  contructor(name) {
+    this.name = name;
+    this.health = 100;
+  }
+
+  static updateAlive(target) {
+    const index = this.alive.indexOf(target);
+
+    if (target.health <= 0) {
+      this.alive.splice(index, 1);
+    } else {
+      this.alive[index].health = target.health;
+    }
```

Comment:
I think this **else** statement is redundant. As **target.health** is already set **Carnivore.bite()** method